packages/lodestar-cli/package.json

```diff
@@ -9,7 +9,10 @@
   },
   "homepage": "https://github.com/ChainSafe/lodestar#readme",
   "main": "lib/index.js",
-  "files": [],
+  "files": [
+    "lib",
+    "bin"
+  ],
```

Comment:
To stay consistent with the rest of packages in the repo you can use
```
"files": [
    "lib/**/*.d.ts",
    "lib/**/*.js",
    "lib/**/*.js.map"
  ],
```
Also, after doing some tests NPM assigns the correct permissions to the symlinks created in `node_modules/.bin/`. So it should not be necessary to push the bin directory to NPM while changing bin to point to `lib/index.js`. The bin directory is still useful but for local scripting without using the `node` prefix

Reply:
ah, right. we can avoid publishing the extra file. I'll update this PR to remove `bin` from `files`, add `lib/index.js` as the bin.